lettuce/django/management/commands/harvest.py

```diff
@@ -34,10 +34,17 @@
 from lettuce.django.server import LettuceServerException
 
 
+DJANGO_VERSION = StrictVersion(django.get_version())
+
+
 class Command(BaseCommand):
     help = u'Run lettuce tests all along installed apps'
     args = '[PATH to feature file or folder]'
-    requires_model_validation = requires_system_checks = False
+
+    if DJANGO_VERSION < StrictVersion('1.9'):
```

Comment:
This clause is unnecessary - we can just use `requires_system_checks` with Django 1.8 as well as Django 1.9.